### PR TITLE
add tutorial for 1026 - Critical Links (en)

### PR DESCRIPTION
**Pull request by Srijond**

### CODE REVIEW

File: en.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+   for(int i = 0; i <bridge.size(); i++)
+   {
+     cout << bridge[i].first << " - " << bridge[i].second << endl;
+}
+
+}
+
+
+return 0;
+}
```
Comment (showrav-ansary) [inline en.md:83]: when you have reached the end of your code, end the code block by again add three ticks after the last line of code but this time you __don't__ need to mention the `programming language`. Like this way:
```md
   //Last line of your code

    ```
```

**Review by showrav-ansary — CHANGES_REQUESTED** [review]

**Comment by showrav-ansary** [comment]
@Srijond Hello, dear contributor. You __have to__ create the `en.md` in a folder.  Create a folder named `1026` and move the `en.md` there. Also if we read your code as `pseudo code` then there is nothing wrong with it. It does give the gist of what to do. However, your code does not run on `LightOJ`, verify there please. Also, you need to format the whole tutorial in markdown. This may help - https://www.markdownguide.org/basic-syntax/

In your solutions, please elaborate more such as:
- What does _tree edges_ mean? Why are they critical?
- What happens to _cross edges_ if they ever occur in some test case?

**Comment by showrav-ansary** [comment]
@Srijond Hello, dear contributor. You __have to__ create the `en.md` in a folder.  Create a folder named `1026` and move the `en.md` there. Also if we read your code as `pseudo code` then there is nothing wrong with it. It does give the gist of what to do. However, your code does not run on `LightOJ`, verify there please. Also, you need to format the whole tutorial in markdown. This may help - https://www.markdownguide.org/basic-syntax/

In your solutions, please elaborate more such as:
- What does _tree edges_ mean? When are they critical and when they are not?
- Would detecting _cycles_ among _nodes_ suggest that they are critical or not?

File: en.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+It is a very basic problem in Bridge(graph) concept ...Actually we need to find how many bridges exists in this graph.Taking input of graph is interesting.First we construct graph and run DFS tree to find back edge and forward edge using discovery time and low time(means there is another path exists).and finally make pair of vertices and print it.
```
[the remark this answers is not in the document]
Comment (Srijond) [reply en.md:1]: OK vaia..I will do it so soon

**Comment by showrav-ansary** [comment]
@Srijond Hello, dear contributor. I have only corrected the `markdown syntax` and some minor phrasing. If you look at it now I think you will understand what I meant by creating a `codeblock` through `markdown` syntax. If you open the `.md` in browser, you will see that a separate block has been created for `codes` and the `syntax` is highlighted in different color.

There is only 1 Issue. You have left `1026` in the root directory. Change the `filename` to `en.md`. `Create` a `folder` named `1026`. `Move` the `en.md` `file` to `1026` `folder`.  

> @Srijond Hello, dear contributor. You **have to** create the `en.md` in a folder. Create a folder named `1026` and move the `en.md` there.

**Comment by showrav-ansary** [comment]
@Srijond thank you for your contribution ❤️  Please check that it has been added and __only__ you are given the __author__ credits when __merging__. I hope you will contribute more 💯

**Pull request merged**